### DISCUSSION TITLE
remnote能添加多选题吗

### DISCUSSION THREAD

**Discussion by staygoldcowboy** [5eagull/RemNote-Chinese-chat-group#54] [你问我答/互相帮助]
尝试使用选择题卡片的时候发现好像只能建立单选的样子，不知道是我没有找到正确的输入方法还是官方没有开发....
我建立选择题用的是::A)
求解答

**Comment by dongrentianyu** [comment]
<img width="1917" height="774" alt="image" src="https://github.com/user-attachments/assets/314da02a-a078-4cb6-8c90-0a1f91d824d7" />

https://help.remnote.com/en/articles/6025481-creating-flashcards

英文文档这里有说明，是可以添加多选题，设置多个正确答案的
